### ISSUE TITLE
Have you a sample usage code ?

### ISSUE THREAD

**Issue by jaysee** [thosoft/google-conversion-tracking#2]
Hi,

this could be usefull : )

thanks

**Comment by Findiglay** [comment]
For iOS conversion tracking, I think we need to add a snippet like this:

`[ACTConversionReporter reportWithConversionID:@"0123456789" label:@"abCDEFG12hIJk3Lm4nO" value:@"0.99" isRepeatable:NO];`

to the AppDelegate.m file as outlined in the documents here - https://developers.google.com/app-conversion-tracking/ios/#add_the_app_download_conversion_snippet

It would be nice if we could install the plugin with a conversionId variable.

**Comment by Findiglay** [comment]
Actually, looking at the js file, it seems we may be able to call conversion reporting manually with the method `GoogleConvertTrack.reportWithConversionId(conversionId, label, value, isRepeatable)`

**Comment by jaysee** [comment]
I'm actually calling it using the following code:

``` javascript
document.addEventListener('deviceready', function() {
    GoogleConvertTrack.reportWithConversionId( "conversionId", "conversionLabel", "conversionValue", "converstionIsRepeatable" );
}, false);
```

But adwords reports nothing... and adwords is shitty with debugging... so i'm stuck... don't know if i'm calling wrong and what is wrong.

NB: For Android, no need to use this plugin as Google automatically inject thoses informations to adWords dashboard.

**Comment by Findiglay** [comment]
@jaysee Yea I'm with you on that.

You can supply callbacks to the method:

```
GoogleConvertTrack.reportWithConversionId('"conversionId", "label",  "value", false,
   function(res) { console.log('reportWithConversionId res:', res); },
   function(err) { console.log('reportWithConversionId err: ', err); }
);
```

My success callback responds with the label (tracking label) which seems to be as intended in the `GoogleConvertTrack.m` implementation. I assume only actual conversions from google ads will be reported however.

**Comment by jaysee** [comment]
Yes you are right, I will use the callbacks to reports success/errors, so i will see some...

Thanks for pointing that.

**Comment by lakshaydulani** [comment]
@jaysee is it working fine for you? thanks

**Comment by jaysee** [comment]
Hello,

So as @Findiglay suggested, I attached the success/fail callback and my debug shows me that the success callback was always called, but people using the adword dashboard wasn't able to confirm some datas (or not), and this is the point where I let the things... and we are not actually using some campaigns so nobody asked me about that, and I can't test for the moment.

**Comment by hoabuiduy** [comment]
I tested. It always success althought no internet conected. Is it issue?
